lib/rules/newline-after-var.js

```diff
@@ -99,13 +116,19 @@ module.exports = function(context) {
 
         // Next statement is not a `var`...
         noNextLineToken = nextToken.loc.start.line > nextLineNum;
-        hasNextLineComment = comments.indexOf(nextLineNum) >= 0;
+        hasNextLineComment = !!commentEndLine[nextLineNum];
```

Comment:
I'm just testing that it's truthy (i.e. a comment exists at `nextLineNum`).  Seemed to be a cleaner check than `(typeof commentEndLine[nextLineNum] !== "undefined")`, but I can understand it's better to be explicit and will revise.